src/azure-cli/azure/cli/command_modules/backup/tests/latest/test_afs_commands.py

```diff
@@ -0,0 +1,444 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import json
+from datetime import datetime, timedelta
+import unittest
+
+from azure.cli.testsdk import ScenarioTest, JMESPathCheckExists, ResourceGroupPreparer, \
+    StorageAccountPreparer
+from .preparers import VaultPreparer, FileSharePreparer, AFSPolicyPreparer, AFSItemPreparer, \
+    AFSRPPreparer, FilePreparer
+
+subscription_id = "da364f0f-307b-41c9-9d47-b7413ec45535"
+unprotected_afs = "clitestafs"
+protected_afs1 = "sarathtestafs"
+protected_afs2 = "afstest1"
+target_afs = "afstest2"
+storage_account = "sarathsa"
+resource_group_name = "sarath-rg"
+vault_name = "sarath-vault"
+
+
+class BackupTests(ScenarioTest, unittest.TestCase):
```

Comment:
Can you try to make your test @record_only? If it is ok, let's make it record_only. If something is wrong, we just don't add record_only.

Reply:
Does this make any difference? As tests are run in record mode only in pipeline. Is this a necessary change?